pbf2json.go

```diff
@@ -549,17 +548,17 @@ func cacheLookupWayNodes(db *leveldb.DB, wayid int64) ([]map[string]string, erro
 
 // decode bytes to a 'latlon' type object
 func bytesToLatLon(data []byte) map[string]string {
-
-	var latlon = make(map[string]string)
+	buf := make([]byte, 0, 8)
+	latlon := make(map[string]string, 4)
 
 	// first 6 bytes are the latitude
-	var latBytes = append([]byte{}, data[0:6]...)
-	var lat64 = math.Float64frombits(binary.BigEndian.Uint64(append(latBytes, []byte{0x0, 0x0}...)))
+	buf = append(buf, data[0:6]...)
```

Comment:
shaving yaks here but this might be slightly more efficient as `copy( buf, data[0:6] )`
without benchmarking it, it's roughly equivalent except it doesn't need to check `len` and `cap`?